sidebars.js

```diff
@@ -32,6 +32,7 @@ const sidebars = {
       type: 'category',
       label: 'Hardware Interfaces',
       items: [
+        'hardware-interfaces/pinouts',
         'hardware-interfaces/external-storage',
```

Comment:
Please update the order based on our documentation map. I believe GPIO should come right after pinouts

Reply:
@greenbreakfast please review the changes.

Reply:
This should be updated **based on our documentation map** 
Please re-order the new articles according to the order listed in the spreadsheet: https://docs.google.com/spreadsheets/d/1ZLPo1iLGbPXOLvLhcrrmB9wG07G94j2TU6RlS2a7ECU/edit#gid=0